Added the Fonts page and updated libraries.md.

8Crafter opened the pull request:
I added a fonts page with documentation for what unicode characters are supported by the in-game Mojangles font.
I also added Prismarine-NBT to NBT libraries list, and added the LevelDB libraries section.

8Crafter committed:
Added the Symbols page.

8Crafter committed:
Update libraries.md
- Added Prismarine-NBT to NBT libraries list.
- Added LevelDB libraries section.

QuazChick commented on docs/text/symbols.md:
```diff
@@ -0,0 +1,368 @@
+
+## Minecraft-Seven-v4.ttf Supported Unicode Characters
+
+This file is used by Ore UI. While referenced by the Ore UI CSS files, it is not actually used. The file can be found in `data/gui/dist/hbui/fonts/Minecraft-Seven-v4-*.ttf`.
```
Minecraft Seven V4 is used in the Ore UI gameplay screens

8Crafter committed:
Update symbols.md

8Crafter committed:
Lots of changes.
- The Symbols page has been renamed to Fonts.
- The character previews on the Fonts page are now 16x instead of 8x.
- The `default8.png` preview on the Fonts page is now displayed at 512x instead of 128x.
- Added the widths for every character on the Fonts page.
- Added a lot more notes to the Fonts page.

QuazChick commented on docs/text/fonts.md:
```diff
@@ -0,0 +1,383 @@
+Affected characters will be labeled as such.
+:::
+
+![](/assets/images/text/symbols/default8/default8.png){style="width: 512px; image-rendering: pixelated;"}
```
This should use a WikiImage component with a `caption` containing the RP/... file path.

8Crafter replied:
Done.

QuazChick commented on docs/text/fonts.md:
```diff
@@ -0,0 +1,383 @@
+
+## In-Game Supported Unicode Characters
+
+These are the characters supported in JSON UI. Their textures are determined by the `default8.png` file (`rp/font/default8.png`).
```
RP should be uppercase
```suggestion
These are the characters supported in JSON UI. Their textures are determined by the `default8.png` file (`RP/font/default8.png`).
```

8Crafter replied:
Done.

QuazChick commented on docs/text/fonts.md:
```diff
@@ -0,0 +1,383 @@
+
+Unicode Range: `U+000D, U+0020-007E, U+00A0-00FF, U+0131, U+0141-0142, U+0152-0153, U+0160-0161, U+0178, U+017D-017E, U+0192, U+02C6-02C7, U+02D8-02DD, U+0394, U+03A9, U+03C0, U+2013-2014, U+2018-201A, U+201C-201E, U+2020-2022, U+2026, U+2030, U+2039-203A, U+2044, U+2074, U+20AC, U+2122, U+2202, U+220F, U+2211-2212, U+221A, U+221E, U+222B, U+2248, U+2260, U+2264-2265, U+25CA, U+FB01-FB02`
+
+## Minecraft-Seven.ttf Supported Unicode Characters
```
Might be better if these headings used the name of the font (e.g. Minecraft Seven V2) rather than the file name.

8Crafter replied:
Done.

8Crafter committed:
Lots of changes to fonts.md.
- Major performance improvements by making glyphs lazy loaded.
- Added Unicode character tables for the other fonts.
- Added 4 more fonts.

QuazChick reviewed (changes requested):
The fonts that have actual font files should be displayed using them rather than including loads of separate SVGs
